### PR TITLE
Update working-with-pipelines.md

### PR DESCRIPTION
**Pull request by t-koman**
Updates for issue https://github.com/kabanero-io/docs/issues/373

### CODE REVIEW

File: publish/working-with-pipelines/working-with-pipelines.md [hunk 1]
```diff
@@ -309,6 +309,76 @@ You can also see pods for the pipeline runs, for which you can specify `oc descr
 
 If the pipeline run was successful, you can see a Docker image in our Docker registry and a pod that’s running your application.
 
+## Transport layer security (TLS) verification for image registry access in pipelines
```
Comment (kvijai82) [inline publish/working-with-pipelines/working-with-pipelines.md]: Can we move this whole section up in the document below the `Using stacks that are published to internal and private registries in pipelines` section?

File: publish/working-with-pipelines/working-with-pipelines.md [hunk 2]
```diff
@@ -309,6 +309,76 @@ You can also see pods for the pipeline runs, for which you can specify `oc descr
...
+
+when accessing image registries to pull and push images, pipelines enable the configuration creation of the cluster resource, [image.config.openshift.io/cluster](https://docs.openshift.com/container-platform/4.2/openshift_images/image-configuration.html#images-configuration-file_image-configuration).  Scenarios are presented of image registries that can be used and the setup steps required.
```
Comment (kvijai82) [inline publish/working-with-pipelines/working-with-pipelines.md]: W in when should be in caps?  Also pipelines don't enable this configuration.  They use the configuration that the user did in the cluster resource.

File: publish/working-with-pipelines/working-with-pipelines.md [hunk 3]
```diff
@@ -309,6 +309,76 @@ You can also see pods for the pipeline runs, for which you can specify `oc descr
...
+
+### Enable TLS verification for image registry access:
+
+#### Private image registries:
+ 
+If you use a private image registry and your registry uses certificates that are signed by trusted CA authorities, no further configuration is needed to enable TLS verification. Review the default truststore on the nodes of your cluster to ensure that you have the CA of your certificate in the list. With a self-signed certificates or a certificate that is signed by a CA, you must ensure that either certificate is listed in the default truststore. Steps are presented to add the CA to the trusted list so that when you run the pipelines, these CA certificates in the list are used.
```
Comment (kvijai82) [inline publish/working-with-pipelines/working-with-pipelines.md]: I think we should update this line `With a self-signed certificates or a certificate that is signed by a CA, you must ensure that either certificate is listed in the default truststore.` to just be `With a self-signed certificates, you must ensure that the CA certificate is add to the appropriate config map as detailed below`.

**Comment by kvijai82** [comment]
While we are updating this guide, can you also add a link to the bottom of the `Creating and updating tasks & pipelines` section to point to this guide (https://github.com/kabanero-io/guides/blob/master/publish/curating-pipelines/curating-pipelines.md) for more detailed instructions.

File: publish/working-with-pipelines/working-with-pipelines.md (hunk 3 above)
Comment (t-koman) [reply publish/working-with-pipelines/working-with-pipelines.md]: Updated as requested

File: publish/working-with-pipelines/working-with-pipelines.md (hunk 2 above)
Comment (t-koman) [reply publish/working-with-pipelines/working-with-pipelines.md]: Updated

File: publish/working-with-pipelines/working-with-pipelines.md (hunk 1 above)
Comment (t-koman) [reply publish/working-with-pipelines/working-with-pipelines.md]: Updated as requested

**Comment by t-koman** [comment]
Kalathar, I made the updates..... let me know if there is anything else required.

**Review by kvijai82 — APPROVED** [review]

File: publish/working-with-pipelines/working-with-pipelines.md [hunk 4]
```diff
@@ -103,6 +103,8 @@ The default tasks and pipelines can be updated by forking the pipelines repo and
 
 Alternatively, you can run the Travis build against a release of your pipelines repo, which also generates the archive file with a `manifest.yaml` file and attaches it to your release.
 
+For more detailed instructions, see [Curating Pipelines](https://github.com/kabanero-io/guides/blob/master/publish/curating-pipelines/curating-pipelines.md)
```
Comment (jantley-ibm) [inline publish/working-with-pipelines/working-with-pipelines.md]: Change the link from:
`[Curating Pipelines](https://github.com/kabanero-io/guides/blob/master/publish/curating-pipelines/curating-pipelines.md)`
to
`[Curating pipelines](../curating-pipelines/curating-pipelines.html)`

**Review by jantley-ibm — COMMENTED** [review]
Thanks Tom, one small change requested

File: publish/working-with-pipelines/working-with-pipelines.md (hunk 4 above)
Comment (t-koman) [reply publish/working-with-pipelines/working-with-pipelines.md]: @jantley-ibm Jeff, the update is made and pushed to the origin

**Review by jantley-ibm — APPROVED** [review]

**Pull request merged**